### ISSUE TITLE
[Feature] Add ability to set a custom system prompt

### ISSUE THREAD

**Issue by sixlive** [tearingItUp786/chatgpt-tui#3]
This is extremely useful if you work with a base character prompt and need to add their prompt to the system

**Comment by sixlive** [comment]
This sounds like a great approach! Looking forward to it!

**Comment by tearingItUp786** [comment]
@sixlive I added a release-candidate version where you can test out the config file. It'll reference the same database as the stable install (which is fine for me). 

Can you give it a spin and let me know if it does what you expect? It suits my needs. 

Commands you'll need to run:

```bash
brew tap tearingitup786/tearingitup786
brew install rc-chatgpt-tui

#edit the config in neovim if you're using it
nvim ~/.chatgpt-tui/config.json

# run the program
rc-chatgpt-tui
```

**Comment by sixlive** [comment]
Works great!!

**Comment by tearingItUp786** [comment]
Fixed in 0.4.0!
